utils/sambanova_endpoint.py

```diff
@@ -995,6 +995,12 @@ class SambaStudioFastCoE(LLM):
 
     model: str = 'llama3-8b'
     """LLM model expert to use"""
+    
+    stream_api: bool = True
+    """use stream api"""
+    
+    stream_options: dict =  {"include_usage": True}
```

Comment:
What does include_usage means here if stream_api is already True?

Reply:
include usage tells the server to sent in the request the token generation metrics jus before the las event